setup.py

```diff
@@ -0,0 +1,51 @@
+
+import os
+import glob
+import platform
+from setuptools import Extension, setup
+from setuptools.command.build import build
+
+cpp_modules = ['ProbeParticle', 'GridUtils', 'fitSpline', 'fitting']
+
+def make():
+    package_path = os.path.dirname(os.path.realpath(__file__))
+    cpp_path = os.path.normpath(package_path + '/ppafm/cpp/')
+    current_directory = os.getcwd()
+    os.chdir (cpp_path)
+    os.system("make clean")
+    os.system("make all")
+    os.chdir (current_directory)
+
+def get_vars_path():
+    if not os.path.exists('C:\Program Files (x86)\Microsoft Visual Studio'):
```

Comment:
Are there better ways to find out whether an executable exists?

Reply:
Probably yes. I expect this to be quite fragile, but I just wanted to make something that works for now.